skimage/data/__init__.py

```diff
@@ -7,45 +7,50 @@
 """
 
 import os as _os
+import warnings
 
 from .. import data_dir
 from ..io import imread, use_plugin
 from .._shared.utils import deprecated
 from ._binary_blobs import binary_blobs
+from .. import img_as_bool
 
 __all__ = ['load',
+           'astronaut',
            'camera',
-           'lena',
-           'text',
            'checkerboard',
-           'coins',
-           'moon',
-           'page',
-           'horse',
-           'clock',
-           'immunohistochemistry',
            'chelsea',
+           'clock',
            'coffee',
+           'coins',
+           'horse',
            'hubble_deep_field',
-           'rocket',
-           'astronaut']
+           'immunohistochemistry',
+           'lena',
```

Comment:
@JDWarner I missed that. You make a good point but I'm happy to have it gone gone. So we can merge this afaic. But I'm agnostic to have it return the ValueError if I'm overridden.